runtime/binding-kafka/src/main/java/io/aklivity/zilla/runtime/binding/kafka/internal/stream/KafkaClientGroupFactory.java

```diff
@@ -413,6 +432,7 @@ public void onDetached(
 
     private MessageConsumer newStream(
         MessageConsumer sender,
+        int initialDelay,
```

Comment:
Given the meaning already associated with `initial` for streams, let's rename `initialDelay` to `rebalanceDelay` in this file.